app/models/book.rb

```diff
@@ -1,3 +1,6 @@
 class Book < ApplicationRecord
-  
+
+  has_and_belongs_to_many :users
```

Comment:
When working with a many-to-many relationship you'll want to use the combination of has_many and has_many through: instead of has_and_belongs_to_many

```ruby
class Student < ApplicationRecord

  has_many :course_students
  has_many :courses, through: course_students
end 
```
This is a better option because has_and_belongs_to_many only ever lets you have the two foreign keys on the join table it creates. You may very likely later need to store some other information on that join table or you might want some logic to live there. With has_and_belongs_to_many you don't have a model and can never add logic.